crates/sui-core/src/authority.rs

```diff
@@ -1141,7 +1141,7 @@ impl AuthorityState {
         certificate: &VerifiedCertificate,
         epoch_store: &Arc<AuthorityPerEpochStore>,
     ) -> SuiResult<TransactionEffects> {
-        let _metrics_guard = if certificate.contains_shared_object() {
+        let _metrics_guard = if certificatens_shared_object() {
```

Comment:
typo?

Reply:
i don't know how this got into the pr, sorry